website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1848,6 +1848,169 @@ $ curl \
 }
 ```
 
+## Assess a workspace
+
+This endpoint assesses a workspace. Only users with admin access are authorized to assess a workspace through a browser session.
```

Comment:
As this is intended to be browser-only. We might want to hold off on official API docs for now.

Reply:
So only add this to the Health page? without any workspaces.mdx copy?

Reply:
Yeah, that might be good. Save this api documentation for when people might eventually interface with this API directly, and mention permissions and API restriction in the health page.